File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/MantenimientoDTO.java
package com.flotix.dto;

public class MantenimientoDTO {

	private String id;
	private String ultimoMantenimiento;
	private String proximoMantenimiento;
	private int kmMantenimiento;
	private String matricula;
	private VehiculoDTO vehiculo;
	private String idTipoMantenimiento;
	private TipoMantenimientoDTO tipoMantenimiento;
	private boolean baja;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getUltimoMantenimiento() {
		return ultimoMantenimiento;
	}

	public void setUltimoMantenimiento(String ultimoMantenimiento) {
		this.ultimoMantenimiento = ultimoMantenimiento;
	}

	public String getProximoMantenimiento() {
		return proximoMantenimiento;
	}

	public void setProximoMantenimiento(String proximoMantenimiento) {
		this.proximoMantenimiento = proximoMantenimiento;
	}

	public int getKmMantenimiento() {
		return kmMantenimiento;
	}

	public void setKmMantenimiento(int kmMantenimiento) {
		this.kmMantenimiento = kmMantenimiento;
	}

	public String getMatricula() {
		return matricula;
	}

	public void setMatricula(String matricula) {
		this.matricula = matricula;
	}

	public VehiculoDTO getVehiculo() {
		return vehiculo;
	}

	public void setVehiculo(VehiculoDTO vehiculo) {
		this.vehiculo = vehiculo;
	}

	public String getIdTipoMantenimiento() {
		return idTipoMantenimiento;
	}

	public void setIdTipoMantenimiento(String idTipoMantenimiento) {
		this.idTipoMantenimiento = idTipoMantenimiento;
	}

	public TipoMantenimientoDTO getTipoMantenimiento() {
		return tipoMantenimiento;
	}

	public void setTipoMantenimiento(TipoMantenimientoDTO tipoMantenimiento) {
		this.tipoMantenimiento = tipoMantenimiento;
	}

	public boolean isBaja() {
		return baja;
	}

	public void setBaja(boolean baja) {
		this.baja = baja;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/MantenimientoDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Flotix2021.ModelDTO
{
    class MantenimientoDTO
    {
		private string _id;

		public string id
		{
			get { return _id; }
			set { _id = value; }
		}

		private string _ultimoMantenimiento;

		public string ultimoMantenimiento
		{
			get { return _ultimoMantenimiento; }
			set { _ultimoMantenimiento = value; }
		}
		
		private string _proximoMantenimiento;

		public string proximoMantenimiento
		{
			get { return _proximoMantenimiento; }
			set { _proximoMantenimiento = value; }
		}
		
		private int _kmMantenimiento;

		public int kmMantenimiento
		{
			get { return _kmMantenimiento; }
			set { _kmMantenimiento = value; }
		}
		
		private string _idVehiculo;

		public string idVehiculo
		{
			get { return _idVehiculo; }
			set { _idVehiculo = value; }
		}
		
		private VehiculoDTO _vehiculo;

		public VehiculoDTO vehiculo
		{
			get { return _vehiculo; }
			set { _vehiculo = value; }
		}
		
		private string _idTipoMantenimiento;

		public string idTipoMantenimiento
		{
			get { return _idTipoMantenimiento; }
			set { _idTipoMantenimiento = value; }
		}
		
		private TipoMantenimientoDTO _tipoMantenimiento;

		public TipoMantenimientoDTO tipoMantenimiento
		{
			get { return _tipoMantenimiento; }
			set { _tipoMantenimiento = value; }
		}
		
		private bool _baja;

		public bool baja
		{
			get { return _baja; }
			set { _baja = value; }
		}
	}
}
